app/views/admin/invoices/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1> Invoices Index </h1>
+
+<% @invoices.each do |invoice| %>
+    <p><%= link_to invoice.id, admin_invoices_path("#{invoice.id}") %></p>
```

Comment:
Not sure if this affects the actual functionality of this link, but you do not need the string interpolation, it can just be admin_invoices_path(invoice.id) :)

Reply:
I was having some trouble with plurals in my show page paths too where it would render the path as "/invoices.34" or whatever the id number is. So if you run into that problem just make the admin_invoices_path admin_invoice_path.